pkgs/tools/text/snippetpixie/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  vala,
+  pkgconfig,
+  wrapGAppsHook,
+  cmake,
+  appstream,
+  desktop-file-utils,
+  python3,
+  libgee,
+  gtk3,
+  sqlite,
+  at-spi2-atk,
+  at-spi2-core,
+  dbus,
+  ibus,
+  json-glib,
+  pantheon
+  }:
+
+stdenv.mkDerivation rec {
+  pname = "snippetpixie";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "bytepixie";
+    repo = pname;
+    rev = version;
+    sha256 = "096xj7n1ypr8ss8mbwd1hyypvmzw5lc0hjlj2d1x8hbjljldfd13";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    vala
+    pkgconfig
+    wrapGAppsHook
+    cmake
```

Comment:
why do we need cmake? I know meson will use it if it can't find a dep through pkgconfig, but we have that too.